### DISCUSSION TITLE
Czy taryfa G12w będzie wspierana?

### DISCUSSION THREAD

**Discussion by sch7dom** [PiotrMachowski/Home-Assistant-custom-components-Tauron-AMIplus#150] [General]
Używam w Tauronie taryfy G12w, chciałbym móc porównywać energię  "w szczycie" i "poza szczytem".  Na stronie Taurona taki podział widzę T1, T2, czy to kwestia jakieiś konfiguracji?

**Comment by PiotrMachowski** [comment]
Integracja wspiera wszystkie dostępne taryfy, wartości dla poszczególnych stref są widoczne w atrybutach

**Reply by prz3m00** [reply]
Cześć - czy mógłbym prosić o wyjaśnienie, co to znaczy, że "wartości dla poszczególnych stref są widoczne w atrybutach"? Pytam dlatego, że zmieniłem taryfę z G11 na G12W i w integracji cały czas widnieje u mnie taryfa G11. W jaki sposób mogę dokonać zmiany na taryfę G12W ? Jestem świeżakiem w HA i Githubie i nieco nie ogarniam...

**Reply by PiotrMachowski** [reply]
Tak wyglądają wartości w atrybutach.

![image](https://github.com/PiotrMachowski/Home-Assistant-custom-components-Tauron-AMIplus/assets/6118709/1548c9f6-706a-4de8-9175-bdf0d31a3eef)

Pokazywana taryfa powinna się zmienić po jakimś czasie, nie ma ona większego znaczenia dla danych pobieranych przez integrację

**Reply by prz3m00** [reply]
OK dzięki za wyjaśnienie, ale właśnie nie wiem w jaki sposób wyświetlić atrybuty, które pokazałeś - jak się w nie wklikać? W którym to jest miejscu ? Nie potrafię tego znaleźć

**Reply by PiotrMachowski** [reply]
Atrybuty są dostępne po wejściu w pop-up encji, poniżej wykresu

**Reply by tomaszgazda** [reply]
Witam

Piotrze czy istnieje jakaś opcja aby wyciągnąć dane z atrybutów i np zrobić  z tego wykres 

Np aby wydobyć  z całośći 2 taryfy s1 i s1 

<img width="829" height="729" alt="image" src="https://github.com/user-attachments/assets/d658e633-a0ef-4174-b225-c37f6c555efe" />

**Reply by PiotrMachowski** [reply]
@tomaszgazda tak, można osiągnąć to template sensorem:
https://www.home-assistant.io/integrations/template

**Reply by tomaszgazda** [reply]
Dzięki za podpowiedź już opanowałem. Dzięki 


> @tomaszgazda tak, można osiągnąć to template sensorem:
> https://www.home-assistant.io/integrations/template
